cisco_aci/tests/test_fabric.py

```diff
@@ -198,11 +198,11 @@ def assert_fabric_port_ingr_metrics(aggregator):
     aggregator.assert_metric(name=metric_name, value=90202911073.0, tags=interface_tags_201_eth2, hostname=hn201)
 
     metric_name = 'cisco_aci.fabric.port.ingr_bytes.unicast'
-    aggregator.assert_metric(name=metric_name, value=50443812.0, tags=interface_tags_101_eth1, hostname=hn101),
-    aggregator.assert_metric(name=metric_name, value=70147142.0, tags=interface_tags_101_eth2, hostname=hn101),
-    aggregator.assert_metric(name=metric_name, value=32704715.0, tags=interface_tags_102_eth1, hostname=hn102),
-    aggregator.assert_metric(name=metric_name, value=23770059.0, tags=interface_tags_102_eth2, hostname=hn102),
-    aggregator.assert_metric(name=metric_name, value=105702610.0, tags=interface_tags_201_eth1, hostname=hn201),
+    (aggregator.assert_metric(name=metric_name, value=50443812.0, tags=interface_tags_101_eth1, hostname=hn101),)
```

Comment:
wut?

Reply:
Oh, good catch! This is because there are commas at the end of each line which should not be there. Ruff understand that we wanted to signify they were tuples and added the parenthesis. This has no effect to be honest but it is super weird.

I can remove the original commas just to clean it up.